libraries/bot-azure/src/test/java/com/microsoft/bot/azure/TranscriptStoreTests.java

```diff
@@ -75,42 +76,38 @@ public static void allTestsInit() throws IOException, InterruptedException {
         assertEmulator();
     }
 
-    @Before
-    public void testInit() {
-        testBlobClient = new BlobContainerClientBuilder()
+    @After
+    public void testCleanup() {
+        BlobContainerClient containerClient = new BlobContainerClientBuilder()
             .connectionString(blobStorageEmulatorConnectionString)
-            .containerName(containerName)
+            .containerName(getContainerName())
             .buildClient();
-        if (!testBlobClient.exists()) {
-            testBlobClient.create();
-        }
-    }
 
-    @After
-    public void testCleanup() {
-        if (testBlobClient.exists()) {
-            testBlobClient.delete();
+        if (containerClient.exists()) {
+            containerClient.delete();
         }
     }
 
     // These tests require Azure Storage Emulator v5.7
     @Test
     public void blobTranscriptParamTest() {
-        Assert.assertThrows(IllegalArgumentException.class, () -> new BlobsTranscriptStore(null, containerName));
+        Assert.assertThrows(IllegalArgumentException.class, () -> new BlobsTranscriptStore(null, getContainerName()));
         Assert.assertThrows(IllegalArgumentException.class, () -> new BlobsTranscriptStore(blobStorageEmulatorConnectionString, null));
-        Assert.assertThrows(IllegalArgumentException.class, () -> new BlobsTranscriptStore(new String(), containerName));
+        Assert.assertThrows(IllegalArgumentException.class, () -> new BlobsTranscriptStore(new String(), getContainerName()));
         Assert.assertThrows(IllegalArgumentException.class, () -> new BlobsTranscriptStore(blobStorageEmulatorConnectionString, new String()));
     }
 
     @Test
     public void transcriptsEmptyTest() {
+        TranscriptStore transcriptStore = getTranscriptStore();
```

Comment:
Can we remove all the `getTranscriptStore()` to be initialized in the property, just like [C#](https://github.com/microsoft/botbuilder-dotnet/blob/main/tests/Microsoft.Bot.Builder.Azure.Tests/BlobsTranscriptStoreTests.cs#L42)?

Reply:
If I'm not wrong, C# assign a function the to the `TranscriptStore` object, and every time it's called execute the function, am I right?

Reply:
I don't see where it's called in each method. I only see that it's assigned or generated in that [line](https://github.com/microsoft/botbuilder-dotnet/blob/main/tests/Microsoft.Bot.Builder.Azure.Tests/BlobsTranscriptStoreTests.cs#L42) and it's used in each test

Reply:
I thought that that assignment was a function, that's why I created the `getTranscriptStore()` method, but I can do what you said

Reply:
I think that it will be better to avoid unnecesary calls, check it